scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/input_model/data/network_groups.yml

```diff
@@ -24,7 +24,19 @@
 
     - name: {{ network_group.name|upper }}
       hostname-suffix: {{ network_group['hostname_suffix'] }}
-{%   if 'MANAGEMENT' in network_group['component_endpoints'] %}
+{%   if ('MANAGEMENT' and 'IPV4')  in network_group['component_endpoints'] %}
+
```

Comment:
Should `hostname` be `True` here as well?

Reply:
I don't think so. The ipv4 network should not be used for the hostname suffix. Only the regular Management and ipv6 management.